crates/rspack_error/src/graphical.rs

```diff
@@ -431,19 +431,23 @@ impl GraphicalReportHandler {
       self.theme.characters.hbar,
     )?;
 
+    let (source_line, source_col) = labels
+      .iter()
+      .find_map(|hl| {
+        lines
+          .iter()
+          .find(|line| line.span_applies(hl) && line.span_line_only(hl))
+          .map(|line| (line.line_number, self.visual_offset(line, hl.offset()) + 1))
```

Comment:
In order to match the result of `Stats`, I think it would be better to use 0-based value for columns.

Reply:
> In order to match the result of `Stats`, I think it would be better to use 0-based value for columns.

Is my understanding correct?

https://github.com/web-infra-dev/rspack/blob/768e43b9ed373511feacabb9dd3e34939bf8badd/crates/rspack_error/src/graphical.rs#L440-L442

Reply:
This is great! Thank you.